standard_test.go

```diff
@@ -2011,3 +2016,33 @@ func TestIsGameOver(t *testing.T) {
 		require.Equal(t, test.Expected, actual)
 	}
 }
+
+type JsonGameInput struct {
+	BoardState BoardState
+	Moves      []SnakeMove
+}
+
+func TestFromJson(t *testing.T) {
+	r := StandardRuleset{}
+
+	files, err := filepath.Glob("tests/standard/*.in.json")
+	require.NoError(t, err)
+
+	for _, file := range files {
+		var input JsonGameInput
+		inputData, err := ioutil.ReadFile(file)
+		require.NoError(t, err)
```

Comment:
I just kinda threw this in for each error-able thing here, is that the right thing to do?